templates/definition/charger/homematic.yaml

```diff
@@ -0,0 +1,65 @@
+template: homematic
+products:
+  - brand: Homematic eQ-3 AG (ELV)
+    description:
+      generic: Homematic IP + BidCoS
```

Comment:
Brauchen wir die Description wirklich? Wenn ich das richtig sehe hat Homematic nur eine Schaltsteckdose in leichten Abwandlungen die aber für Doku und Webseite erstmal irrelevant sind. Würde vorschlagen die Description erstmal raus zu nehmen.